modules/nodes-clusters-cgroups-2.adoc

```diff
@@ -0,0 +1,194 @@
+// Module included in the following assemblies:
+//
+// * nodes/clusters/nodes-cluster-cgroups-2.adoc
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+ifeval::["{context}" == "nodes-cluster-cgroups-2"]
+:nodes:
+endif::[]
+ifeval::["{context}" == "post-install-cluster-tasks"]
+:post:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="nodes-clusters-cgroups-2_{context}"]
+= Configuring Linux cgroup v2
+
+ifdef::post[]
+You can enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control group version 2] (cgroup v2) in your cluster by editing the `node.config` object. Enabling cgroup v2 in {product-title} disables all cgroups version 1 controllers and hierarchies in your cluster. cgroup v1 is enabled by default.
+
+cgroup v2 is the next version of the Linux cgroup API. cgroup v2 offers several improvements over cgroup v1, including a unified hierarchy, safer sub-tree delegation, new features such as link:https://www.kernel.org/doc/html/latest/accounting/psi.html[Pressure Stall Information], and enhanced resource management and isolation. 
+
+[IMPORTANT]
+====
+{product-title} cgroups version 2 support is a Technology Preview feature only. Technology Preview features
```

Comment:
Can this be a snippet `include` too?

Reply:
@adellape I could not get the include to work in the ifdef. Do you know how to make it work?